### PR TITLE
Lessons/7

### LINKED ISSUE #1: Update master branch
Your work is split into several branches. Make an effort to update your master branch. Rebase if necessary

### PR DESCRIPTION
**Pull request by yandiro**
Mergin Lesson7 to Master to start 'second step' from here.

### CODE REVIEW

File: app/components/library-item-form.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Component from '@ember/component';
+import { action } from '@ember/object';
+
+export default class LibraryItemFormComponent extends Component {
+  // pass an action to override
+  handleClick = this.args?.handleClick;
```
Comment (tpetrone) [inline app/components/library-item-form.js:6]: why did you need to do this?

**Review by tpetrone — APPROVED** [review]
- always leave a `\n` (nem line) at file's end
- remove test files since they are not being used

File: app/components/library-item-form.js (hunk 1 above)
Comment (yandiro) [reply app/components/library-item-form.js:6]: To overwrite the function in the inner component with the one passed in from the outer component.

Here:
https://github.com/zoltan-nz/library-app/blob/master/app/components/library-item-form.js
